File: 1518-water-bottles/1518-water-bottles.java
class Solution {
    public int numWaterBottles(int numBottles, int numExchange) {
        int drink=numBottles;
        while(true){
            int remg=numBottles%numExchange;
            int exchange=numBottles/numExchange;
            if(exchange==0)
                return drink;
            drink+=exchange;
            numBottles=remg+exchange;
        }
    }
}

File: 1518-water-bottles/1518-water-bottles.py
class Solution(object):
    def numWaterBottles(self, numBottles, numExchange):
        """
        :type numBottles: int
        :type numExchange: int
        :rtype: int
        """
        drink=numBottles
        while True:
            remg=numBottles%numExchange
            exchange=numBottles//numExchange
            if exchange==0:
                return drink
            drink=drink+exchange
            numBottles=remg+exchange
        